lib/http/request.ts

```diff
@@ -176,14 +176,14 @@ export function httpRequest(sdk: OktaAuthHttpInterface, options: RequestOptions)
     // Running fetch after short timeout fixes this issue.
     waitForAwakenDocument = () => {
       const timeSinceDocumentIsVisible = Date.now() - dateDocumentBecameVisible;
-      if (timeSinceDocumentIsVisible < IOS_PAGE_AWAKEN_TIMEOUT) {
+      if (timeSinceDocumentIsVisible < pollDelay!) {
```

Comment:
Any idea why `!` is required here? Does TS not know pollDelay is defined?

Reply:
Fixed TS issue with `pollDelay = sdk.options.pollDelay ?? 0;`